client-hc/src/main/java/com/graphhopper/api/GraphHopperWeb.java

```diff
@@ -64,8 +64,8 @@
     private boolean calcPoints = true;
     private boolean elevation = false;
     private String optimize = "false";
-    boolean postRequest = true;
-    int maxUnzippedLength = 1000;
+    private boolean postRequest = true;
+    private int maxUnzippedLength = 1000;
```

Comment:
Had to add a setter here, since we are now no longer testing this in the same package, do you think thats ok?